client-js/app/elements/labrad-registry.html

```diff
@@ -170,41 +172,75 @@
-                      on-keys-pressed="cursorMove"></iron-a11y-keys>
-      <iron-a11y-keys target="[[target]]" keys="right enter"
-                      on-keys-pressed="cursorTraverse"></iron-a11y-keys>
-      <iron-a11y-keys target="[[target]]" keys="left backspace"
-                      on-keys-pressed="cursorBack"></iron-a11y-keys>
-      <!-- Dialog Controls -->
-      <iron-a11y-keys target="[[target]]" keys="esc"
-                      on-keys-pressed="dialogCancel"></iron-a11y-keys>
-      <iron-a11y-keys target="[[target]]" keys="shift+enter"
-                      on-keys-pressed="dialogSubmit"></iron-a11y-keys>
-      <!-- Search Controls -->
-      <iron-a11y-keys target="[[target]]" keys="enter"
-                      on-keys-pressed="searchSubmit"></iron-a11y-keys>
-      <table>
-        <tbody is="selectable-table" id="combinedList" selected="{{selectedIdx}}">
-          <template id="fullList" is="dom-repeat" items="{{listItems}}" filter="filterFunc">
-            <tr name="{{item.name}}"
+  <div id="outer">
+    <!-- Dialog Controls -->
+    <iron-a11y-keys target="[[target]]" keys="esc"
+                    on-keys-pressed="dialogCancel"></iron-a11y-keys>
+    <iron-a11y-keys target="[[target]]" keys="shift+enter enter"
+                    on-keys-pressed="dialogSubmit"></iron-a11y-keys>
+    <!-- Cursor Controls -->
+    <iron-a11y-keys target="[[target]]" keys="up down"
+                    on-keys-pressed="cursorMove"></iron-a11y-keys>
+    <iron-a11y-keys target="[[target]]" keys="right enter"
+                    on-keys-pressed="cursorTraverse"></iron-a11y-keys>
+    <iron-a11y-keys target="[[target]]" keys="left backspace"
+                    on-keys-pressed="cursorBack"></iron-a11y-keys>
+    <!-- Search Controls -->
+    <iron-a11y-keys target="[[target]]" keys="enter"
+                    on-keys-pressed="searchSubmit"></iron-a11y-keys>
+    <paper-header-panel id="left-column">
+      <div class="paper-header" id="buttons">
+        <paper-toolbar>
+          <paper-button on-click="newKeyClicked">
+            <iron-icon icon="create"></iron-icon>
+            new key
+          </paper-button>
+          <paper-button on-click="newFolderClicked">
+            <iron-icon icon="folder-open"></iron-icon>
+            new dir
+          </paper-button>
+          <paper-button on-click="copyClicked" disabled="[[!selected]]">
+            <iron-icon icon="content-copy"></iron-icon>
+            copy
+          </paper-button>
+          <paper-button on-click="renameClicked" disabled="[[!selected]]">
+            <iron-icon icon="code"></iron-icon>
+            rename
+          </paper-button>
+          <paper-button on-click="deleteClicked" disabled="[[!selected]]">
+            <iron-icon icon="delete"></iron-icon>
+            delete
+          </paper-button>
+          <div id="search-bar">
+            <paper-input id="search" value="{{filterText::input}}"
+              noLabelFloat>
```

Comment:
The hanging indents here are a bit inconsistent, sometimes using 4 spaces, here 2, and sometimes aligning with attribute on the first line. I think 4 spaces or aligned would be fine, but we should be consistent (at least within a single .html if not throughout the codebase).